src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2452,8 +2470,22 @@ subroutine CNOffsetLitterfall (num_soilp, filter_soilp, &
 
             if (offset_counter(p) == dt) then
                t1 = 1.0_r8 / dt
-               leafc_to_litter(p)  = t1 * leafc(p)  + cpool_to_leafc(p)
+               !leafc_to_litter(p)  = t1 * leafc(p)  + cpool_to_leafc(p)
                frootc_to_litter(p) = t1 * frootc(p) + cpool_to_frootc(p)
+               
+               ! Y. Cheng, cut 70% AGB for switchgrass and Miscanthus and move to grainc_to_food
+               if (ivt(p)==nmiscanthus .or. ivt(p)==nirrig_miscanthus .or. ivt(p)==nswitchgrass .or. ivt(p)==nirrig_switchgrass) then
+               	   leafc_to_litter(p)  = t1 * leafc(p)*0.3  + cpool_to_leafc(p)
+               	   grainc_to_food(p) =grainc_to_food(p)+ t1 * leafc(p) * 0.7 
+               	   grainn_to_food(p) =grainn_to_food(p)+ t1 * leafn(p) * 0.7
+               	   
+!                	   leafc_to_litter(p)  = (t1 * leafc(p)  + cpool_to_leafc(p))*0.3
+!                	   grainc_to_food(p) =grainc_to_food(p)+ (t1 * leafc(p)  + cpool_to_leafc(p))*0.7
+!                	   grainn_to_food(p) =grainn_to_food(p)+ t1 * leafn(p) * 0.7
+               else
```

Comment:
It looks like you calculate grainc_to_food and grainn_to_food here for miscanthus and switchgrass, but I don't see where it's still calculated for other crops (it was removed below with the comment that it was calculated previously). I would have thought that the original calculation for each would be included in the "else" statement here. I'm new to looking at/reviewing code in GitHub, so sorry if I missed seeing where this was done.